plugins/inputs/ping/ping.go

```diff
@@ -172,6 +179,10 @@ func (p *Ping) nativePing(destination string) (*pingStats, error) {
 		pinger.SetNetwork("ip6")
 	}
 
+	if p.Method == "native" {
+		pinger.Size = p.Size
```

Comment:
Doesn't seem like `p.Size` is being initialized anywhere, wouldn't this be 0 if this isn't configured?

Reply:
`p.Size` will indeed default to 0, but that's the size of the _data bytes_. With the ICMP header included, that will form an ICMP packet of 8 bytes.

Reply:
Thank you for the explanation, that makes sense. Because the README for the size flag is `56`, I wonder if we also default to `56` instead of `0` as well?

It looks like go-ping originally defaulted to `16` so that could be an appropriate default as well, but `56` probably makes more sense.

Reply:
Sure! Let's default to 56.